.travis/before_install_osx.sh

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
I'm not really interested in an macOS script. We test on Linux only and I don't want to change that in this PR.